src/test/java/org/fundacionjala/coding/yerel/EANValidatorTest.java

```diff
@@ -0,0 +1,59 @@
+package org.fundacionjala.coding.yerel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+/**
+ * this a unit test for EANValidator.
+ */
+public class EANValidatorTest {
+
+    public static final int EXPRESULT = 72;
+    private EANValidator eanValidator;
+
+    /**
+     * this is begin a instants method.
+     */
+    @Before
+    public void setUp() {
+        eanValidator = new EANValidator();
+    }
+
+    /**
+     * this test verify 4003301018398 is a true.
+     */
+    @Test
+    public void validator() {
+        assertTrue(eanValidator.validator("4003301018398"));
+    }
+
+
+    /**
+     * this test verify 400330101839 add is true.
+     */
+    @Test
+    public void sumNumbersCode() {
+        assertEquals(EXPRESULT, eanValidator.sumNumbersCode("400330101839"));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "400330101839" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)